test/src/eks_blueprints_e2e_test.go

```diff
@@ -245,13 +239,14 @@ func eksAddonValidation(t *testing.T, eksClusterName string, awsRegion string) {
 	/****************************************************************************/
 	/*TEST: Verify the total number of nodes running
 	/****************************************************************************/
-	nodes, err := k8sclient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
-	if err != nil {
-		t.Errorf("Error getting EKS nodes: %v", err)
-	}
-	t.Run("MATCH_TOTAL_EKS_WORKER_NODES", func(t *testing.T) {
-		assert.Equal(t, expectedEKSWorkerNodes, len(nodes.Items))
-	})
+	// nodes, err := k8sclient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+	// if err != nil {
+	// 	t.Errorf("Error getting EKS nodes: %v", err)
+	// }
+	// t.Run("MATCH_TOTAL_EKS_WORKER_NODES", func(t *testing.T) {
+	// 	assert.Equal(t, expectedEKSWorkerNodes, len(nodes.Items))
+	// 	assert.Greater(t, len(nodes.Items), 0)
```

Comment:
Not a great test, node count will fluctate. Left it as commented out but happy to remove entirely as well